pandunia-lekse.csv

```diff
@@ -601,6 +610,7 @@ duwa|||tur: dua, swa: dua, may: doa, ara: دُعَاء‏ /duʿāʾ/|pray||rezar
 duwe||||prayer||rezo (oración)|||||||||||||preĝo|rukous|modlitwa
 duxe|||fra: douche, deu:Dusche, tur:duş, rus:душ (duš), ara:(duš), spa:por:ducha|shower||ducha|||||||||||||duŝo|suihku|prysznic
 e|||fra:et, por:e, zho:和 (hé)|and|et|y|e|||和|||||||||kaj|ja|i, oraz
+efete|||spa:efecto, por:efeito, fra:effet, eng:effect, deu:Effekt, rus:эффект (effekt), tur:efekt, may:efek|effect (result)|effet|efecto|efeito|эффект||影响|効果||||||||||
```

Comment:
There is already "rezulte".

Reply:
I just missed "rezulte". We can use that.